include/multigrid.h

```diff
@@ -245,6 +245,9 @@ namespace quda {
     /** Coarse temporary vector */
     ColorSpinorField *tmp2_coarse;
 
+    /** Kahler-Dirac Xinv */
+    cudaGaugeField *xInvKD;
```

Comment:
Why is this a `cudaGaugeField` and not just `GaugeField`?

Reply:
To a point, no good reason, but it's more a declaration that the optimized KD operator acts more like the current fine ops than a coarse op. The current fine ops contain `cudaGaugeField` or `cudaCloverField` as opposed to the more general types, reflecting (for now) that the fine ops can only be applied to device fields, though I know that's going away soon.

In any case, I can update this.